app/Resources/views/base.html.twig

```diff
@@ -154,6 +154,15 @@
             <script>
                 $(document).ready(function() {
                     hljs.initHighlightingOnLoad();
+
+                    $('form[data-confirmation-message]').on('submit', function (event) {
```

Comment:
my second comment about event delegation has not been taken into account though.